src/app/web-purchase-thank-you/page.tsx

```diff
@@ -0,0 +1,89 @@
+import {
+  StyledThankYouColumnContent,
+  StyledThankYouColumns,
+  StyledGradientBottom,
+  StyledTextContent,
+  StyledThankYouSection,
+  StyledSectionCTA,
+} from './thank-you.styles';
+import { StyledDisplayDevice } from '../frida/frida.styles';
+
+import { Container } from '@/components/elements/container';
+import { Heading } from '@/components/elements/heading';
+import Stack from '@/components/elements/stack';
+import { Text } from '@/components/elements/text';
+import { ThemedImage } from '@/components/elements/themed-image';
+import { LandingLayout } from '@/components/layout/landing-layout';
+import { ContentCard } from '@/components/modules/content-card';
+import { DownloadButton } from '@/components/modules/download-button';
+import { NEWSLETTER_URLS } from '@/components/modules/newsletter/newsletter.values';
+
+export default function WebPurchaseThankYouPage() {
```

Comment:
@RBoraurito, Please add the capture purchase to posthog, check here: https://github.com/httptoolkit/httptoolkit-website/blob/fde8f6ed4304b4d81a0d57d55b5915383fb2c63d/src/pages/web-purchase-thank-you.jsx#L65